doc/support_matrix.yml

```
@@ -494,6 +494,15 @@ boards:
       user_usb: not_currently_supported
       ethernet_over_usb: not_currently_supported
 
+  heltec-v3:
+    name: Heltec V3 ESP32S3 with SX1262 LoRa-Node 
```

This seems to be the name the manufacturer uses:

```suggestion
    name: Heltec WiFi LoRa 32 V3
```

---

https://heltec.org/project/wifi-lora-32-v3/